1pass

```diff
@@ -198,7 +198,6 @@ signin()
     se=$("$GPG" -d -q "$secret")
     if [ "${use_totp}" == "1" ]; then
         local ot
-        local totp
```

Comment:
Why is this removed?

Reply:
The local variable has been removed to access the location of the totp file. By having it local, bash is unable to find the path to the totp file.

Sorry for the late reply. Exams happened

Reply:
Awesome -- thanks for the response!